pkg/packagekit/package_fpm.go

```diff
@@ -134,16 +134,16 @@ func PackageFPM(ctx context.Context, w io.Writer, po *PackageOptions, fpmOpts ..
 		fpmCommand = append(fpmCommand, "--before-remove", filepath.Join("/pkgscripts", "prerm"))
 	}
 
-	dockerArgs := []string{
+	args := []string{
 		"run", "--rm",
-		"-v", fmt.Sprintf("%s:/pkgsrc", po.Root),
-		"-v", fmt.Sprintf("%s:/pkgscripts", po.Scripts),
-		"-v", fmt.Sprintf("%s:/out", outputPathDir),
+		"-v", fmt.Sprintf("%s:/pkgsrc:Z", po.Root),
```

Comment:
Hrm... Docker has several warnings about this:

> If you use selinux you can add the z or Z options to modify the selinux label of the host file or directory being mounted into the container. This affects the file or directory on the host machine itself and can have consequences outside of the scope of Docker.
> 
> * The z option indicates that the bind mount content is shared among multiple containers.
> * The Z option indicates that the bind mount content is private and unshared.
>
> Use extreme caution with these options. Bind-mounting a system directory such as /home or /usr with the Z option renders your host machine inoperable and you may need to relabel the host machine files by hand.

I think I understand the why of this, but it makes me kinda unsure.

Reply:
Interesting. My interpretation of the warning is: don't attach important system directories as Docker volumes as it may allow the Docker volume to manipulate the data.

If it makes you feel better, I can add the `:Z` only for podman users. The reason this works in Docker is that by default, Docker runs as root. podman normally executes as a regular user (rootless mode). Docker also has a rootless mode, but nobody uses it :)